nav2_bt_navigator/src/navigators/navigate_through_poses.cpp

```diff
@@ -213,17 +231,24 @@ bool
 NavigateThroughPosesNavigator::initializeGoalPoses(ActionT::Goal::ConstSharedPtr goal)
 {
   Goals goal_poses = goal->poses;
+  int i = 0;
   for (auto & goal_pose : goal_poses) {
     if (!nav2_util::transformPoseInTargetFrame(
         goal_pose, goal_pose, *feedback_utils_.tf, feedback_utils_.global_frame,
         feedback_utils_.transform_tolerance))
     {
-      RCLCPP_ERROR(
-        logger_,
-        "Failed to transform a goal pose provided with frame_id '%s' to the global frame '%s'.",
-        goal_pose.header.frame_id.c_str(), feedback_utils_.global_frame.c_str());
+      current_error_code_ = ActionT::Result::TF_ERROR;
```

Comment:
I don't think any of this actually gets sent anywhere since this is in the goal acceptance part of the action, not the goal processing.

Reply:
So I think it could be removed?

Reply:
Keeping as discussed. Renamed to GOAL_TRANSFORMATION_ERROR